howso/details.amlg

```diff
@@ -426,9 +435,35 @@
 			(call !ComputeCaseMDA)
 		)
 
+		(if (and (!= 0 (size (get details "prediction_stats"))) (not (get details "get_prediction_stats")))
```

Comment:
`get_prediction_stats` was scrapped as a detail I think?

Reply:
yes, forgot to remove